opencodelists/settings.py

```diff
@@ -162,6 +164,11 @@ def get_env_var(name):
     },
 ]
 
+if TAKE_SCREENSHOTS:
```

Comment:
What's the reasoning for conditionally including the context processor rather than always including it? It means the `if TAKE_SCREENSHOTS` is duplicated in two places (the settings.py and in the context processor itself). It might be clearer just to always include it, unless I've missed something?

Reply:
Thanks for spotting that.

I don't think there is a reason, other than me probably misreading how the `IN_PRODUCTION` context processor worked, and being overzealous in trying to add a toggle.